.gflows/libs/job_docker_build.lib.yml

```
@@ -19,7 +19,11 @@
 #@ end
 ---
 #@ def _image_build_job(component, sections, needs=["version"], tag_build=None, job_name=None, push=True, environment_variables=None):
-#@ steps = _image_build_job_steps(component, sections.cache_registry, tag_build, job_name,  push)
+#@ build_versioned_image= False
+#@ if (hasattr(component,"build_versioned_image")):
```

you can use shorter syntax like 
```suggestion
#@ build_versioned_image=getattr(component,"build_versioned_image",False))
```
this line will look nice as a new method in "configuration" library.